gap/attr.gi

```diff
@@ -1582,3 +1582,125 @@ function(gr)
   od;
   return fail;
 end);
+
+InstallMethod(VertexConnectivity,
+"for a digraph",
+[IsDigraph],
+function(digraph)
+  local edges, edmondskarp, k, kappa, kappas, mindegv, Nv, outdegs, relabel,
+  sdigraph, set1v, x, y;
+
+  if DigraphNrVertices(digraph) = 0 then
+    return 0;
+  fi;
+
+  sdigraph := DigraphSymmetricClosure(DigraphRemoveAllMultipleEdges(digraph));
```

Comment:
I don't think this line should be necessary, since `digraph` anyway contains all the information about its symmetric closure.